lib/vfscore/automount.c

```diff
@@ -87,8 +184,8 @@ UK_LIBPARAM_PARAM_ARR_ALIAS(fstab, &vfscore_fstab, charp,
 /**
  * Expected command-line argument format:
  *	vfs.fstab=[
- *		"<src_dev>:<mntpoint>:<fsdriver>[:<flags>:<opts>]"
- *		"<src_dev>:<mntpoint>:<fsdriver>[:<flags>:<opts>]"
+ *		"<src_dev>:<mntpoint>:<fsdriver>[:<flags>:<opts>:<ukopts>]"
+ *		"<src_dev>:<mntpoint>:<fsdriver>[:<flags>:<opts>:<ukopts>]"
```

Comment:
We can also briefly mention that `flags`, `opts`, and `ukopts` can be empty as well.

Reply:
I do mention this below

Reply:
👍🏻